lib/esp32.toit

```diff
@@ -137,11 +137,21 @@ rtc_user_bytes -> ByteArray:
   #primitive.esp32.rtc_user_bytes
 
 /**
-Produces (as a system message) a report over the usage of memory at the OS level.
+Sends (as a system message) a report over the usage of memory at the OS level.
 */
 memory_page_report -> none:
   report := memory_page_report_
   send_trace_message report
 
 memory_page_report_ -> ByteArray:
   #primitive.esp32.memory_page_report
+
+/**
+Sends (as a system message) a detailed report over the usage of memory at the OS level.
+*/
+dump_heap -> none:
+  report := dump_heap_ 300
```

Comment:
Can we get rid of the versions that write to serial/flash from C code? They are unused?

Reply:
I want to revive them too.